main.go

```diff
@@ -156,3 +156,26 @@ func registerCapabilities(port string, log logger.Handler) {
 		log.Info(err.Error())
 	}
 }
+
+func registerDynamicCapabilities(port string, log logger.Handler) {
+	registerWorkloads(port, log)
+	//Start the ticker
+	const reRegisterAfter = 24
```

Comment:
Will you add this behavior to the user-facing Meshery Docs on Adapters?

Reply:
yeah, after this gets merged. Will sure do